sdk/communication/Azure.Communication.CallingServer/tests/CallConnection/CallConnectionLiveTests.cs

```diff
@@ -175,6 +175,57 @@ public async Task RunCreateAddMuteGetParticipntRemoveHangupScenarioTests()
             }
         }
 
+        [Test]
+        public async Task RunCreateAddRemoveFromDefaultAudioGroupAddToDefaultAudioGroupRemoveHangupScenarioTests()
+        {
+            if (SkipCallingServerInteractionLiveTests)
+                Assert.Ignore("Skip callingserver interaction live tests flag is on.");
+
+            CallingServerClient client = CreateInstrumentedCallingServerClientWithConnectionString();
+
+            CallConnection callConnection = new CallConnection(null, null, null);
+
+            try
+            {
+                // Establish a Call
+                callConnection = await CreateCallConnectionOperation(client).ConfigureAwait(false);
+                await WaitForOperationCompletion().ConfigureAwait(false);
+
+                string userId = GetUserId(USER_IDENTIFIER);
+
+                // Add Participant
+                AddParticipantResult addParticipantResult = await AddParticipantOperation(callConnection, userId).ConfigureAwait(false);
+                Assert.NotNull(addParticipantResult);
+                await WaitForOperationCompletion().ConfigureAwait(false);
+
+                // Remove Participant from Default Audio Group
+                await RemoveParticipantFromDefaultAudioGroupOperation(callConnection, userId).ConfigureAwait(false);
+
+                // Add Participant To Default Audio Group
+                var addParticipantToDefaultAudioGroupResult = await AddParticipantToDefaultAudioGroupOperation(callConnection, userId).ConfigureAwait(false);
```

Comment:
we do not need to wait for this. also, there is no return value for adding to default routing group